chapters/synchronous.tex

```diff
@@ -1103,59 +1113,60 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 For example, a tool could handle events in the same way as for a usual simulation -- but only check them at the time associated with clock-ticks.
 
 Alternatively, relations might be interpreted literally, so that events are no longer triggered (in order that the time for an integration step is always the same, as needed for hard real-time requirements).
 However, even if relations do not generate events, when-clauses and operators \lstinline!edge! and \lstinline!change! should behave as normal.
 \end{nonnormative}
 
-From the view of the clocked partition, the continuous-time partition is discretized and the discretized continuous-time variables have only a value at a clock tick.
-Therefore, such a partition is handled in the same way as any other clocked partition.
-Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized continuous-time partition with other partitions.
-Hereby, a discretized continuous-time partition is seen as a clocked partition.
-
+From the viewpoint of other partitions, the discretized continuous-time variables only have values at clock ticks (internally it may be more complicated, see \cref{solver-methods}).
+Therefore, outside the discretized partions themselves, they are treated similarly to discrete-time partitions.
+Especially, operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate signals of the discretized partition with other partitions.
 
-\subsection{Clocked Discrete-Time and Discretized Continuous-Time}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}
+\subsection{Discrete-time and Discretized Partitions}\label{clocked-discrete-time-and-clocked-discretized-continuous-time-partition}\label{clocked-and-discretized-partition}
 
 Additionally to the variability of expressions defined in \cref{variability-of-expressions}, an orthogonal concept \willintroduce{clocked variability} is defined in this section.
 If not explicitly stated otherwise, an expression with a variability such as \emph{continuous-time} or \emph{discrete-time} means that the expression is inside a partition that is not associated to a clock.
 If an expression is present in a partition that is not a continuous-time partition, it is a \firstuse[clocked!expression]{clocked expression}\index{expression!clocked} and has \firstuse[clocked!variability]{clocked variability}\index{expression variability!clocked}.
 
-After sub-clock inferencing, see \cref{sub-clock-inferencing}, every partition that is associated to a clock has to be categorized as \willintroduce{clocked discrete-time} or \willintroduce{clocked discretized continuous-time} partition.
+After sub-clock inferencing, see \cref{sub-clock-inferencing}, every sub-clock partition that is associated with a clock has to be categorized as a \willintroduce{discrete-time} or a \willintroduce{discretized} partition.
```

Comment:
To avoid misinterpreting _partition_ as base partition, it might be safer to remove (in order to avoid repeating the lengthy _sub-clock partition_):
```suggestion
After sub-clock inferencing, see \cref{sub-clock-inferencing}, every sub-clock partition that is associated with a clock has to be categorized as \willintroduce{discrete-time} or \willintroduce{discretized}.
```